### DISCUSSION TITLE
安装报错

### DISCUSSION THREAD

**Discussion by AWH-ai** [pgEdge/cli#34] [General]
in use  "./nodectl install pgedge -U postgres -P 123456 -d postgres"  Command installation error prompt;
WARNING! Change to EL9 or Ubuntu 22.04 for advanced functionality.
########### Installing pgedge ###############
['pgedge']

Get:1 https://oscg-io-download.s3.amazonaws.com/REPO pgedge-2-4

ERROR: HTTP Error 403: Forbidden
url=https://oscg-io-download.s3.amazonaws.com/REPO, file=pgedge-2-4.tar.bz2

Why did this problem arise and how should we handle it

**Comment by luss** [comment]
likely that you have a proxy that needs to be downloaded thru.  Can you confiorm?
